src/Lykke.Job.Messages.Services/Email/EmailGenerator.cs

```diff
@@ -475,6 +475,18 @@ private string GetFailedTransactionBody(string transactionId, string[] clientIds
             return await _templateGenerator.GenerateAsync(partnerId, "SwiftCashoutDeclined", templateVm);
         }
 
+        public async Task<EmailMessage> GenerateRegistrationVerifyEmailMsg(string partnerId, RegistrationEmailVerifyData messageData)
+        {
+            var templateVm = new RegistrationEmailVerifyData
+            {
+                Code = messageData.Code,
+                Url = messageData.Url,
+                Year = DateTime.UtcNow.Year.ToString()
+            };
+
+            return await _templateGenerator.GenerateAsync(partnerId, "EmailVerificationCode", templateVm);

```

Comment:
No need to use async/await you can do just return _templateGenerator.GenerateAsync